packages/@tinacms/fields/src/components/Select.tsx

```diff
@@ -46,11 +53,20 @@ export const Select: React.FC<SelectProps> = ({ input, field, options }) => {
         {...input}
       >
         {selectOptions ? (
-          selectOptions.map(option => (
-            <option value={option} key={option}>
-              {option}
-            </option>
-          ))
+          selectOptions.map((option: Option) => {
```

Comment:
Great! 

My only ask is to make this a two step mapping:

```ts
selectOptions
  .map(toProps)
  .map(toComponent)
```

where `toMap` looks something like:

```ts
function toProps(option: Option) {
  if (typeof option === 'object') return option
  return { value: option, label: option }
}
```

Reply:
Great suggestion! Should be resolved now. Should I update the docs too?